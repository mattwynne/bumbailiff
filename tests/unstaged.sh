#!/bin/bash
# TODO: put these on path
source "../../../test_helpers.sh"
mkdir -p "git" && cd "git" || exit
create_git_repo
git commit --quiet -m "Initial commit" --allow-empty
touch file.js
git add .
git commit --quiet --allow-empty -m "Add file.js"
echo """# TODO: rewrite in ruby
console.log('yo')
""" >file.js
bumbailiff
echo $?
