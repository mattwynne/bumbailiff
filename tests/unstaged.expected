[1mfile.js:1[0m
[0;32m0 days old[0;37m Not committed yet (Daisy Developer)[0m
TODO: rewrite in ruby

ðŸŒ± [0;32m0 TODO-days accumulated (max allowance: 14 ðŸ’µ).[0m
0
